src/tree/mod.rs

```diff
@@ -49,6 +49,7 @@ pub struct RatchetTree {
     /// All nodes in the tree.
     /// Note that these only hold public values.
     /// Private values are stored in the `private_tree`.
+    /// FIXME: this must not be public
```

Comment:
This will also be fixed in the tree rewrite. I just created #423, so maybe we can just mention that in place of the existing FIXMEs.